packages/paste-website/src/pages/foundations/elevation/index.mdx

```diff
@@ -0,0 +1,221 @@
+
+The majority of elements will be at this level in their resting states, however an element on this level can move to a higher level when interacted with.
+
+For example: a Button might be ‘Level 0’ in its resting state, but ‘Level 1’ when hovered.
+
+### Level 1
+
+Elements at this level require subtle differentiation from the surface behind them without adding too much distance. These elements are either presented by default or triggered within the composition but typically do not open on top of other elements.
+
+There are 2 different ways to communicate an element's elevation in level 1:
+
+1. **Use an accent shadow** for elements that need to be distinct from the surrounding elements but not highlighted.
+
+<ResponsiveImage src={AccentShadow} alt="mock of side panel component with 'shadow elevation 05' as box shadow" />
+
+<ApplyingLevel1ImgCaption />
+
+2. **Use a tonal difference** when elements need to be highlighted as well as distinct from the background.
+
+<ResponsiveImage src={TonalChange} alt="a mock of callout component with 'color background body elevation' token as background color that makes it subtly more prominent" />
+
+<CalloutExampleImgCaption />
+
+### Level 2
+
+Elements that rest at this level are commonly distinctly separated from the background and dynamically overlap content layers creating visible distance while still being bound to Level 0 elements.
+
+There are 2 different ways to communicate an element's elevation in level 2:
+
+1. **Use a tonal difference with an accent shadow** when an element needs to be highlighted and prominent but does not overlap content by default. **Combine this with a soft sharp shadow** when the element overlaps other content and extra distance is necessary to make it distinct from the background.
+
+<ResponsiveImage src={Level2AccentChange} alt="mock of a top bar component that uses 'color background weak elevation' as background color and 'shadow elevation bottom 05' as box shadow" />
+
+<ApplyLevel2ImgCaption />
+
+2. **Use a soft and sharp shadow** for elements that have temporary prominence or subtly float above the content.
+
+*Interactive states like vertical or horizontal scroll can dynamically use a soft, sharp shadow to indicate that there is overflown content.
+
+<ResponsiveImage src={Level2RisenShadow} alt="a mock of popover component that uses 'shadow elevation 10' as box shadow" />
+
+<PopoverExampleImgCaption />
+
+### Level 3
+
+Elements that rest at this level, float above all other levels, and tend to contain crucial actions or information that temporarily becomes the top hierarchy of the surface.
+
+1. **Use a large, soft shadow with edges**, for elements that are at the top priority and require user focus, but still allow the user to interact with the surrounding content.
+
+For elements with decorative or semantic use of color (background or borders), use a large soft shadow without edges.
```

Comment:
```suggestion
1. **Use a large, soft shadow**, for elements that are at the top priority and require user focus, but still allow the user to interact with the surrounding content.

For elements that already have their own border, use `$shadow-elevation-05`. Otherwise, use `$shadow-elevation-top-05` to define its edges against the background.
```

@roxanagomez I feel like we could say this section more directly ^. Thoughts?